src/progress-bar/SongProgress.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { BrowserSessionContext } from '../BrowserSessionProvider';
+import { useState, useContext, useEffect } from 'react';
+import ProgressBar from 'ink-progress-bar';
+import { Box } from 'ink';
+
+const SongProgress = () => {
+
+  const session = useContext(BrowserSessionContext);
+  const [currentProgress, setCurrentProgress] = useState(session.ProgressUpdates.currentProgress);
+
+  useEffect(() => {
+    session.ProgressUpdates.subscribe((currentProgress) => {
+      setCurrentProgress(currentProgress);
+    });
+
+  }, []);
+
+  return (
+    <Box marginBottom={0.5} borderStyle="round">
+      <ProgressBar
+        color="blue"
+        percent={currentProgress.currentTime / currentProgress.currentDuration}
```

Comment:
I think we should consider destructuring these out of currentProgress earlier, and storing the quotient in a variable, that way later if we want to include an actual display of the time adjacent to the progressbar, we can easily add it. Doesn't have to be done now though.

Reply:
Are you meaning calculating the % completed and a string formatted like '3:00:15:30' and passing those?